Source/Core/Core/ConfigManager.cpp

```diff
@@ -929,18 +930,8 @@ bool SConfig::SetPathsAndGameMetadata(const BootParameters& boot)
 
 DiscIO::Region SConfig::GetFallbackRegion()
```

Comment:
You could probably get rid of this function entirely and just call `Config::Get(Config::MAIN_FALLBACK_REGION)` instead.

Reply:
It's used for GameCube and Wii region fallbacks, so I'm not sure I want to change it from one function being used twice to using a configuration var in two different places. If it is indeed preferred, I can make that change though.

Reply:
> flagrama: JosJuice, do you definitely want me to make the change mentioned in your comment?
JosJuice: It's not too important
JosJuice: Though if you don't want to make the change, then I would suggest making the function's implementation one line instead of two lines
JosJuice: You can return the result of Config::Get directly without storing it to a variable

Went with the alternative given.